ClientDesktop/App.xaml

```diff
@@ -2,7 +2,7 @@
              xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:local="clr-namespace:ClientDesktop"
-             StartupUri="MainWindow.xaml">
+             StartupUri="SearchWindow.xaml">
```

Comment:
Let's keep some naming conventions in place. It's called User/Index in the MVCClient, so let's rename this to "UserIndex". Even better, see if you can put these "window" files into folders (something like User folder, and IndexWindow.xaml inside).